cli/src/commands/working-groups/verifyValidator.ts

```diff
@@ -0,0 +1,38 @@
+import { flags } from '@oclif/command'
+import WorkingGroupsCommandBase from '../../base/WorkingGroupsCommandBase'
+import { WorkingGroups } from '../../Types'
+import { u64 } from '@polkadot/types'
+
+export default class VerifyValidatorAccountCommand extends WorkingGroupsCommandBase {
+  static description = 'Membership lead/worker verifies validator membership profile'
+  static flags = {
+    memberId: flags.string({
+      required: true,
+      description: 'Membership ID of the validator to verify',
+    }),
+
+    ...WorkingGroupsCommandBase.flags,
+  }
+
+  async run(): Promise<void> {
+    const { memberId } = this.parse(VerifyValidatorAccountCommand).flags
+
+    const members = await this.getApi().groupMembers(WorkingGroups.Membership)
+    const lead = await this.getApi().groupLead(WorkingGroups.Membership)
+
+    const membersRows = members.filter((m) => {
+      const arrayValue: u64[] = [m.memberId]
+      return this.getRequiredWorkerContext('MemberController', arrayValue)
+    })
```

Comment:
As I mentioned you don't need explicitly get all the workers & lead and then find the actual worker. Please refer to https://github.com/chrlschwb/joystream/blob/d8518bb7ad015052b080e793fce297a34c763abf/cli/src/commands/working-groups/leaveRole.ts example command to see how to use `getRequiredWorkerContext` method properly

Reply:
The reason why I'm looping through all workers is to determine if the caller is a membership WG worker or a worker in a different WG.

Reply:
You can simply change the default working group by executing the following function at the start of the command 

```
await this.setPreservedState({ defaultWorkingGroup: WorkingGroups.Membership })
```

Then running `this.getRequiredWorkerContext()` would check the worker in the context of the Membership working group